www/src/views/builder/index.ejs

```diff
@@ -321,23 +321,36 @@
 											<option value="3">Shield slot only</option>
 										</select>
 									</div>
+                                </div>
+								<div class="col-auto" ng-if="currentItem.slot == 13 || currentItem.slot == 2">
+									<div class="input-group input-group-sm">
+										<div class="input-group-prepend">
+										</div>
+										<!-- Button trigger modal -->
+										<button type="button" class="btn btn-primary" data-toggle="modal" ng-if="!isRuneCrafting" ng-click="runeCraftButtonClick()" data-target="#runeCraftCharmModal">
+											Runecraft Customizer
+										</button>
+										<button type="button" class="btn btn-primary" data-toggle="modal" ng-if="isRuneCrafting" ng-click="runeCraftButtonClick()" data-target="#runeCraftCharmModal">
```

Comment:
Are these `data-toggle` and `data-target` attributes outdated? I thought you were just setting a boolean flag and basing an `ng-if` on that value.

Reply:
The two buttons are there to toggle between the rune customizer and regular items.

Reply:
Yea but you don't need the data-toggle and data-target attributes, do you? Since it doesn't open a modal anymore?